Documentation/guides/messages/xa1023.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Xamarin.Android warning XA1023
+description: XA1023 warning code
+ms.date: 05/18/2020
+---
+# Xamarin.Android warning XA1023
+
+## Example messages
+
+```
+warning XA1023: Using the DX Dex Compiler is deprecated. Please update `$(AndroidDexTool)` to `d8`.
+```
+
+## Issue
+
+Google has deprecated the DX Dex Compiler in favor of the [D8 Dex
+Compiler][d8]. On [February 1, 2021][dx], DX will no longer be a part
+of Android SDK or Android Studio.
+
+[d8]: https://developer.android.com/studio/command-line/d8
+[dx]: https://android-developers.googleblog.com/2020/02/the-path-to-dx-deprecation.html
+
+## Solution
+
+Update the `$(AndroidDexTool)` MSBuild property to `d8` to select the
+D8 Dex Compiler. This property corresponds to the **Dex compiler**
+setting in the Visual Studio project properties pages. Alternatively,
+you can remove `<AndroidDexTool>` from your `.csproj` file to rely on
+the default value.
```

Comment:
Possible rewording to make the "you" implicit (and one little change to italics to prepare the doc for integration on docs.microsoft.com):

```
Alternatively, remove `<AndroidDexTool>` from the _.csproj_ file to let the
build select the default value `d8`.
```

No strong preference on this on my side. My thought is this makes sentence more similar to the first "Update the ..." sentence, though that probably has both pros and cons.